pages/linux/lvresize.md

```diff
@@ -1,15 +1,20 @@
 # lvresize
 
 > Change the size of a logical volume.
+> More information: <https://man7.org/linux/man-pages/man8/lvresize.8.html>.
 
-- Change a volume's size to 120GB:
+- Change the size of a logical volume to 120GB:
 
-`lvresize -L {{120G}} {{logical_volume}}`
+`lvresize --size {{120G}} {{volume_group}}/{{logical_volume}}`
 
-- Reduce a volume's size by 120GB as well as the underlying filesystem:
+- Extend the size of a logical volume by 120GB as well as the underlying filesystem:
```

Comment:
```suggestion
- Extend the size of a logical volume as well as the underlying filesystem by 120GB:
```

Reply:
just committed your suggestions the proper way. thanks!